site_tracking.js

```diff
@@ -1,31 +1,17 @@
 if (typeof(php_data.ac_settings.site_tracking) != "undefined" && php_data.ac_settings.site_tracking == "1") {
+	(function(e,t,o,n,p,r,i){e.prismGlobalObjectAlias=n;e.pgo=e.pgo||function(){(e.pgo.q=e.pgo.q||[]).push(arguments)};e.pgo.l=(new Date).getTime();r=t.createElement("script");r.src=o;r.async=true;i=t.getElementsByTagName("script")[0];i.parentNode.insertBefore(r,i)})(window,document,"http://prism.app-us1.com/prism.js","pgo");
 
-	// Set to false if opt-in required
-	var trackByDefault = php_data.ac_settings.site_tracking_default;
+	pgo('setAccount', php_data.ac_settings.tracking_actid);
+	pgo('setTrackByDefault', php_data.ac_settings.site_tracking_default == "1");
+	pgo('process');
 
 	function acEnableTracking() {
 		var expiration = new Date(new Date().getTime() + 1000 * 60 * 60 * 24 * 30);
 		document.cookie = "ac_enable_tracking=1; expires= " + expiration + "; path=/";
-		acTrackVisit();
+		pgo('process', 'allowTracking');
 	}
 
-	function acTrackVisit() {
-		var trackcmp_email = php_data.user_email;
-		var trackcmp = document.createElement("script");
-		trackcmp.async = true;
-		trackcmp.type = 'text/javascript';
-		trackcmp.src = '//trackcmp.net/visit?actid=' + php_data.ac_settings.tracking_actid + '&e=' + encodeURIComponent(trackcmp_email) + '&r=' + encodeURIComponent(document.referrer) + '&u=' + encodeURIComponent(window.location.href);
```

Comment:
Just one question - since this function is being removed, where do we pass into Prism the `actid`, `email`, `referer`, etc? I don't have any idea what Prism is, so it's likely I just don't understand how it works. 👍

Reply:
@mthommes No worries. All of that is handled by Prism now, which then reaches out to Trackcmp. I do think I shouldn't have deleted the email part, I will add that back in.

Reply:
@mthommes Can you take a look at the latest commit?